### ISSUE TITLE
$type is required

### ISSUE THREAD

**Issue by robodude666** [crcn/sift.js#195]
The following code in typescript:

```ts
const data = [3, 4, 5, 6, 7]
const f = sift({ $gt: 5 })
console.log(data.filter(f))
```
produces the following error:

```
Argument of type '{ $gt: number; }' is not assignable to parameter of type 'Query'.
  Property '$type' is missing in type '{ $gt: number; }' but required in type 'Query'.ts(2345)
core.d.ts(23, 5): '$type' is declared here.
```

A workaround would be to:

```ts
sift({ $gt: 5, $type: Number})
// or
sift({ $gt: 5 } as Query)
```

Is `$type` intended to be required, or should it be optional -- like all other operations?

**Comment by pavel-voronin** [comment]
Looks like a mistyping

**Comment by crcn** [comment]
Fixed & published to npm as `v13.0.4`. Thanks for catching this!
